src/main/java/uk/ac/ebi/subs/repository/model/templates/SingleRefCapture.java

```diff
@@ -0,0 +1,53 @@
+package uk.ac.ebi.subs.repository.model.templates;
+
+import lombok.Builder;
+import lombok.Data;
+import org.json.JSONObject;
+
+import java.util.Collections;
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+public class SingleRefCapture implements Capture {
+
+    private String refKey;
+    private String displayName;
+    private boolean required;
+
+    @Override
+    public Capture copy() {
+        return this.toBuilder().build();
+    }
+
+    @Override
+    public int capture(int position, List<String> headers, List<String> values, JSONObject document) {
+        String value = values.get(position);
+
+        if (value != null) {
+            addRef(value,document);
+        }
+        return ++position;
+    }
+
+    @Override
+    public int map(int position, List<Capture> captures, List<String> headers) {
+
+        this.setCaptureInList(position,captures,headers.get(position));
+
+        return ++position;
+    }
+
+    @Override
+    public List<String> expectedColumnHeaders() {
+        return Collections.emptyList();
```

Comment:
Is this always an empty list?

Reply:
Yes. Some of 'capture' classes take multiple columns, after primary the named column (e.g. attributes have the main value column, + optional units and terms columns.

I'll move this function to be the interface default, as there's no point replicating it all over the place